conf/messages.cy

```diff
@@ -202,16 +202,9 @@ tariff.declaration.type.text.2 = Dewch o hyd i fanylion am {0}.
 tariff.declaration.type.linkText.2 = wneud datganiad allforio llawn (yn agor tab newydd)
```

Comment:
Are you sure you still need lines 199-203 and 206-207?

Reply:
Removed unused messages, few still in use.